spec/hbw/features/tasks/claim_task_mock.yml

```diff
@@ -0,0 +1,476 @@
+## Structure of file:
+# <method>:
+#   <url>:
+#     - params: <body-1 or querystring-1>
+#       response: <response-1>
+#     - params: <body-2 or querystring-2>
+#       response: <response-2>
+
+get:
+  tasks: {}
+  user:
+    - params: "email=c.johnson@example.com"
+      response:
+        data:
+          - id:         'demo'
+            firstName:  'Demo'
+            lastName:   'Demo'
+            url:        'http://bpm.example.com'
+            email:      'c.johnson@example.com'
+            pictureUrl: null
+        total: 1
+        start: 0
+        sort: 'id'
+        order: 'asc'
+        size: 1
+
+  # process definitions
+  process-definition:
+    - params: ""
+      response:
+        - &default_process-definition
+          id: "testProcess:469:222221"
+          key: "supportRequestProcess"
+          version: 469
+          name: "Test name"
+          description: null
+          tenantId: null
+          deploymentId: "222221"
+          resource: "testProcess.bpmn"
+          diagram: null
+          category: "http://bpmn.io/schema/bpmn"
+          suspended: false
+          versionTag: null
+          historyTimeToLive: null
+        - <<: *default_process-definition
+          id: "testProcess:469:222222"
+          deploymentId: "222222"
+        - <<: *default_process-definition
+          id: "testProcess:469:222223"
+          deploymentId: "222223"
+
+  # form definitions
+  task/111111/deployed-form: &form_with_everything
+    - params: ""
+      response: |
+        form:
+          name: Test form with options
+          css_class: col-xs-12 col-sm-6 col-md-5 col-lg-4
+          fields:
+          - name: group1
+            type: group
+            label: Choose option
+            css_class: col-xs-12
+            fields:
+            - name: homsOrderDataStatic
+              type: static
```

Comment:
All the claiming tests don't care about task form content.

Reply:
I'll make issue about cutting off such redundant things from tests